nixos/modules/services/admin/docuum.nix

```diff
@@ -0,0 +1,37 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.docuum;
+in
+{
+  options.services.docuum = {
+    enable = mkEnableOption "docuum daemon";
+
+    threshold = mkOption {
+      description = "Threshold for deletion";
```

Comment:
Add more details on how the threshold format looks. Add another plausible example.